examples/workers/pages/index.ts

```diff
@@ -0,0 +1,27 @@
+import { Router } from 'worktop';
+import * as cfw from 'worktop/cfw';
+
+import type { Context } from 'worktop';
+import type { Durable } from 'worktop/cfw.durable';
+
+interface MyContext extends Context {
+	bindings: {
+		ASSETS: Durable.Namespace;
```

Comment:
I think this needs to be a durable object in order to compile, assuming `"@cloudflare/workers-types": "3.3.1"`
```suggestion
		ASSETS: Durable.Object;
```

Reply:
No this is correct as `Durable.Namespace` – worktop includes its own types and doesn't rely on the cloudflare/workers-types package. A namespace loads a `Durable.Object`